### PR TITLE
docs(elements): update images and summaries

### LINKED ISSUE #1008: [docs] "All elements" docs need review/cleanup
Please review sections and ensure docs are complete.

- ["All elements" page at ux.redhat.com](https://ux.redhat.com/elements/)
- [XD file](https://xd.adobe.com/view/5bdbf729-fee4-49ab-b06c-683dc4d64216-0e88/)
- [Google Doc](https://docs.google.com/document/d/1udnh5wDxuzDV1NDY1QBg1tJMenTOSnntXU8x46t9Sec/edit?usp=sharing)

Tag @coreyvickery for final design review.

### PR DESCRIPTION
**Pull request by eyevana**
Closes #1008

## What I did

1. Updated all the images and summaries for the `/elements` landing page according to the design [spec](https://docs.google.com/document/d/1udnh5wDxuzDV1NDY1QBg1tJMenTOSnntXU8x46t9Sec/edit)
2.  Modified the `example` shortcode to allow for absolute paths in its image src

## Testing Instructions

1. Review the linked spec and verify that it matches the deploy [preview](https://deploy-preview-1007--red-hat-design-system.netlify.app/elements)

### CODE REVIEW

**Review by nikkimk — CHANGES_REQUESTED** [review]
Looking good so far with a couple changes on the DP:

- Elements should be sorted by and alias not tagName
- Elements should be displaying alias not tagName ("Cta" should be "Call to Action", "Stat" should be "Statistic", etc.)
- Card's summary does not match the Doc
- Pagination has no summary displaying
- Foundations section at the bottom should be replaced with Patterns

**Comment by eyevana** [comment]
@nikkimk Thanks for the review. I think I've addressed all the issues you identified. Can you take another looksy, pleaseee? <3

File: elements/rh-pagination/rh-pagination.ts [hunk 1]
```diff
@@ -24,6 +24,9 @@ const L2 = html`
 
 /**
  * Pagination
```
Comment (bennypowers) [inline elements/rh-pagination/rh-pagination.ts:26]: should have a more verbose description here

Comment (eyevana) [reply elements/rh-pagination/rh-pagination.ts:26]: For sure. This will be addressed in https://github.com/RedHat-UX/red-hat-design-system/pull/993.

**Comment by coreyvickery** [comment]
@eyevana Some very small edits:

- Add `Breadcrumb (coming soon)` card and thumbnail, but do not link it anywhere
- Remove the gray border around the `Dialog` card
- Add `Footnote (coming soon)` card and thumbnail, but do not link it anywhere
- Change `Navigation primary` to `Navigation (primary)`
- Change `Subnav` to `Subnavigation`
- Change the text under the `Make a request` heading to:

To request a new element or if updates need to be made to an existing element, [create a GitHub issue](https://github.com/RedHat-UX/red-hat-design-system/issues/new/choose).

The link above should open a new tab or window.

**Comment by coreyvickery** [comment]
@eyevana Hey, I do not see the `(coming soon)` element cards and text as described in my comment above.

**Comment by eyevana** [comment]
@coreyvickery yep, I'm working on that last piece now. I'll ping you here when it's ready for another round of reviews.

**Comment by eyevana** [comment]
@coreyvickery I added Breadcrumb and Footnote to /elements as coming soon. Do you also want me to include them in the side nav? For example, "Breadcrumb (coming soon)" with no link?

**Comment by coreyvickery** [comment]
> @coreyvickery I added Breadcrumb and Footnote to /elements as coming soon. Do you also want me to include them in the side nav? For example, "Breadcrumb (coming soon)" with no link?

@eyevana No thanks.

**Comment by eyevana** [comment]
@bennypowers @nikkimk can you take a look at how I handled the "coming soon" elements and let me know what you think? I stubbed them under the `/elements` directory so they could be included and sorted under the `elementDocs` collection, and then I just added some conditional logic in the templates to concatenate the title with "(coming soon)".

**Comment by coreyvickery** [comment]
@eyevana Sorry, I forgot to mention that we are removing `Subnavigation` for now.

**Comment by eyevana** [comment]
@coreyvickery exclude completely or just mark it as coming soon?

**Comment by coreyvickery** [comment]
@eyevana Yes, kill it. Or create a new issue to kill it website-wide.

**Comment by eyevana** [comment]
@coreyvickery ready for you 👁️ ❤️

**Comment by eyevana** [comment]
@nikkimk thank you!

**Review by bennypowers — CHANGES_REQUESTED** [review]
pls move images back to elements/*/docs/screenshot.png, as they can be used on other listings, not only on ux-dot

**Review by bennypowers — APPROVED** [review]

**Comment by eyevana** [comment]
> @eyevana looks like there are still some conflicts to resolve

Done! :)

**Comment by eyevana** [comment]
> > @eyevana looks like there are still some conflicts to resolve
> 
> Done! :)

Actually need to make one more change

**Pull request merged**